aci/resource_aci_fvepselector.go

```diff
@@ -0,0 +1,205 @@
+package aci
+
+
+import (
+	"fmt"
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAciEndpointSecurityGroupSelector() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAciEndpointSecurityGroupSelectorCreate,
+		Update: resourceAciEndpointSecurityGroupSelectorUpdate,
+		Read:   resourceAciEndpointSecurityGroupSelectorRead,
+		Delete: resourceAciEndpointSecurityGroupSelectorDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciEndpointSecurityGroupSelectorImport,
+		},
+
+		SchemaVersion: 1,
+
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"endpoint_security_group_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+		},
+
+			"annotation": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				Description: "Mo doc not defined in techpub!!!",
+		},
```

Comment:
Annotation field is not needed here, because I added annotation in the base attribute schema.